### PR TITLE
Some Runspec improvements

### PR DESCRIPTION
**Pull request by akva2**
- split out Phase enum
- remove unnecessary includes

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this opm-simulators=4349 please

**Comment by akva2** [comment]
jenkins build this opm-simulators=4349 please

**Comment by akva2** [comment]
jenkins build this opm-simulators=4349 please

File: opm/input/eclipse/Schedule/Well/Well.hpp [hunk 1]
```diff
@@ -30,10 +30,8 @@
 #include <utility>
 #include <vector>
 
-#include <stddef.h>
```
Comment (bska) [inline opm/input/eclipse/Schedule/Well/Well.hpp:33]: Note that by removing `<stddef.h>`, which I absolutely support, we must also add a namespace qualifier&mdash;i.e., `std::`&mdash;to the current "bare" `size_t` uses in this file.  There is no guarantee that `<cstddef>` makes `size_t` available in the global namespace.

**Review by bska — COMMENTED** [review]
This looks good for the most part.  We need to be a little careful about removing `<stddef.h>` though since there are a number of uses of unqualified `size_t` types scattered about the files.

File: opm/input/eclipse/Schedule/Well/Well.hpp (hunk 1 above)
Comment (akva2) [reply opm/input/eclipse/Schedule/Well/Well.hpp:33]: fixed.

**Comment by akva2** [comment]
jenkins build this opm-simulators=4349 please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  I'll merge this and its downstream companion into the master branch.

**Pull request merged**